### PR TITLE
feat(frontend): update favicon and page titles tm-295

### PR DESCRIPTION
**Pull request by Vfivew**

### CODE REVIEW

**Review by GEOFARL — APPROVED** [review]

File: apps/frontend/src/libs/hooks/use-app-titile/use-app-title.hooks.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { useEffect } from "~/libs/hooks/hooks.js";
+
+const useAppTitle = (title: string): void => {
+	useEffect(() => {
+		document.title = title;
```
Comment (what1s1ove) [inline apps/frontend/src/libs/hooks/use-app-titile/use-app-title.hooks.ts:5]: do not duplicate enum values "TrackMates" should be the default

then if you provide something you need to write the logic for it `${title ? `${title} | ` : ''}TrackMates`

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: apps/frontend/src/libs/hooks/use-app-titile/use-app-title.hooks.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { useEffect } from "~/libs/hooks/hooks.js";
+
+const useAppTitle = (title?: string): void => {
+	useEffect(() => {
+		document.title = title ? `${title} | TrackMates` : "TrackMates";
```
Comment (what1s1ove) [inline apps/frontend/src/libs/hooks/use-app-titile/use-app-title.hooks.ts:5]: do not duplicate `TrackMates`

File: apps/frontend/index.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 	<head>
 		<meta charset="UTF-8" />
-		<link rel="icon" type="image/svg+xml" href="/vite.svg" />
+		<link rel="icon" type="image/svg+xml" href="/assets/favicon.ico" />
```
Comment (what1s1ove) [inline apps/frontend/index.html:5]: `.ico` is not enough 
https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: apps/frontend/index.html (hunk 3 above)
Comment (Vfivew) [reply apps/frontend/index.html:5]: Vlad, here I simply added the missing part. In fact, I have a standard set for icons (I'll attach a screenshot). Or do you want 48х48 and 72х72 and others?
Thi
?"

Comment (what1s1ove) [reply apps/frontend/index.html:5]: Got it, thank you

**Review by Ann-Maksay — APPROVED** [review]

**Review by Sytnikovakatya — APPROVED** [review]

File: apps/frontend/src/libs/hooks/use-app-titile/use-app-title.hooks.ts (hunk 2 above)
Comment (Vfivew) [reply apps/frontend/src/libs/hooks/use-app-titile/use-app-title.hooks.ts]: This way ${title ? ${title} |  : ''}TrackMates not work. Linter not allowed it.
So i did `document.title = (title ? title + " | " : "") + "TrackMates";`
I hope its good solution.

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**